### PR TITLE
TS: Fix type declarations of renderer classes.

### PR DESCRIPTION
**Pull request by taphos**
Fixes for internal types, makes implementing custom Renderer easier

### CODE REVIEW

File: src/renderers/webgl/WebGLBackground.d.ts [hunk 1]
```diff
@@ -3,17 +3,17 @@ import { Color } from '../../math/Color';
...
-	getClearColor(): void;
-	setClearColor( color: Color, alpha: any ): void;
-	getClearAlpha(): void;
-	setClearAlpha( alpha: any ): void;
-	render( renderList: WebGLRenderLists, scene: Scene, camera: any, forceClear: any ): void;
+	getClearColor(): Color;
+	setClearColor( color: Color, alpha: number ): void;
+	getClearAlpha(): number;
+	setClearAlpha( alpha: number ): void;
+	render( renderList: WebGLRenderList, scene: Scene, camera: any, forceClear: any ): void;
```
Comment (Mugen87) [inline src/renderers/webgl/WebGLBackground.d.ts:17]: While you're at it^^:
```ts
render( renderList: WebGLRenderList, scene: Scene, camera: Camera, forceClear: boolean ): void;
```

File: src/renderers/webgl/WebGLCapabilities.d.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ export class WebGLCapabilities {
 		parameters: WebGLCapabilitiesParameters
 	);
 
-	readonly isWebGL2: true;
+	readonly isWebGL2: boolean;
```
Comment (Mugen87) [inline src/renderers/webgl/WebGLCapabilities.d.ts:14]: The lib usually set `true` for all `is*` properties. Hence, I would undo this particular change.

File: src/renderers/webgl/WebGLBackground.d.ts (hunk 1 above)
Comment (Mugen87) [reply src/renderers/webgl/WebGLBackground.d.ts]: You can also assign `Camera` as the type of `camera`. It's still `any`.

Comment (taphos) [reply src/renderers/webgl/WebGLBackground.d.ts]: done

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**